### ISSUE TITLE
Chrome Extension Website Incorrect

### ISSUE THREAD

**Issue by RickCogley** [ZenHubIO/support#179]
The linked developer website in the Chrome Extension is .com not .io, and should be updated to the correct URL.

**Comment by azenMatt** [comment]
Thanks for the feedback @RickCogley - I have scanned all over the extension and the chromestore and can not find the .com link - do you mind letting me know where exactly this is and I will get it fixed!

**Comment by RickCogley** [comment]
How odd, when I wrote that, I had just clicked a link and got a 404, because it was listed as zenhub.com <http://zenhub.com/> not zenhub.io. But now that I look again, I cannot find it. Perhaps I was caching an older version, or something. 

Sorry for the trouble. If I see it again, I’ll let you know. Apologies. 
Regards
Rick 

--
Rick Cogley 
Mobile: 090-9959-5452 (Outside Japan, remove "0". +81-90-...)
